packages/router-plugin/src/composed.ts

```diff
@@ -0,0 +1,25 @@
+import { unpluginRouterGeneratorFactory } from './router-generator'
+import { unpluginRouterCodeSplitterFactory } from './code-splitter'
+
+import type { Config } from './config'
+import type { UnpluginFactory } from 'unplugin'
+
+export const unpluginRouterComposedFactory: UnpluginFactory<
+  Partial<Config> | undefined
+> = (options = {}, meta) => {
+  const generatorResult = unpluginRouterGeneratorFactory(options, meta)
+
+  const generator = Array.isArray(generatorResult)
+    ? generatorResult
+    : [generatorResult]
+
+  const codeSplitterResult = unpluginRouterCodeSplitterFactory(options, meta)
+  const codeSplitter =
+    meta.framework === 'rspack'
```

Comment:
why does rspack behave differently here?

Reply:
Yeah, it doesn't support the `resolveId` and `transform` calls.

Need to do more investigation into whether I can get this same functionality using their compiler hooks. If not, I'll add an asterisk next to experimental code-splitting not working.